rpc_util.go

```diff
@@ -365,6 +366,36 @@ func (o MaxRecvMsgSizeCallOption) before(c *callInfo) error {
 }
 func (o MaxRecvMsgSizeCallOption) after(*callInfo, *csAttempt) {}
 
+// CallAuthority returns a CallOption which sets the authority to override the
+// `:authority` pseudoheader in a RPC. If this is set, the RPC will use only
```

Comment:
This is the only place where we use the term `pseudoheader`. Either use it everywhere (like in the dosctrings of the `AuthorityValidator` interface), or don't use it here. Thanks.

Reply:
How about something more simpler and direct

"CallAuthority creates a CallOption that sets the HTTP/2 :authority header of an RPC to the specified value."